docs/CHANGELOG.asciidoc

```diff
@@ -65,6 +66,317 @@ Review important information about the {kib} 8.x releases.
 * <<release-notes-8.0.0-alpha1>>
 
 --
+[[release-notes-8.14.0]]
+== {kib} 8.14.0
+
+For information about the {kib} 8.14.0 release, review the following information.
+
+[float]
+[[breaking-changes-8.14.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and performance.
+Before you upgrade to 8.14.0, review the breaking changes, then mitigate the impact to your application.
+
+[discrete]
+[[breaking-182074]]
+.Renamed the an advanced setting to enable {esql}.
```

Comment:
This text confuses me `the an advanced setting` seems wrong, no?

Reply:
Oh yikes that is confusing, removing the 'the'.